cookbooks/fastscape_eroding_box.prm

```diff
@@ -0,0 +1,172 @@
+# At the top, we define the number of space dimensions we would like to
+# work in:
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 2.5e5
+set Maximum time step                      = 50000 #5e6
+set Output directory                       = eroding_box
+
+set Pressure normalization                 = no #surface
+set Surface pressure                       = 0
+
+# A box which has one half raised by 1 km.
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 100e3
+    set Y extent = 100e3
+    set Z extent = 25e3
+    set X repetitions = 4
+    set Y repetitions = 4
+  end
+  
+  subsection Initial topography model
+    set Model name = function
+    subsection Function
+      set Function expression = if(y>20e3 && y<80e3 && x>20e3 && x<80e3, 2500, 0)          
+    end
+  end
+end
+
+# We do not consider temperature in this setup
+subsection Initial temperature model
+  set Model name = function
+
+  subsection Function
+    set Function expression = 0
+  end
+end
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top
+  set List of model names = initial temperature
+end
+
+# X boundaries are periodic, and the rest are tangential.
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, front, back, left, right
+end
+
+# Here we setup the top boundary with fastscape.
+subsection Mesh deformation
+  set Mesh deformation boundary indicators = top : fastscape
+  
+    subsection Fastscape
+      # As the highest resolution at the surface is 4 in the mesh refiniment function, we set this the same.
+      set Surface resolution = 4
+
+      # Because we only have 1 level of refinement at the surface we set this to 0.
```

Comment:
```suggestion
      # Because we have the same level of refinement at the surface we set this to 0.
```

Reply:
This earlier comment from @anne-glerum is not yet addressed